### ISSUE TITLE
[text] Line height for wrapped text

### ISSUE THREAD

**Issue by 956MB** [raysan5/raylib#1992]
Is there not a method for adjusting the line height of wrapped text? This would be a nice and simple addition to have in the text module.

Example change to the `DrawTextRec` method:

`DrawTextRec(Font font, const char *text, Rectangle rec, float fontSize, float spacing, bool wordWrap, float lineHeight, Color tint);`

The `lineHeight` float could be used when `wordWrap` is true, and you have text long enough to wrap. I'm pretty sure I've checked everywhere, I don't think I've missed a method for it.

**Comment by raysan5** [comment]
@956MB Actually that function has been removed from raylib and moved to examples to allow users to configure it as desired.

**Comment by KyrietS** [comment]
Currently, the implementation of `DrawTextEx` has this:

```c
        if (codepoint == '\n')
        {
            // NOTE: Fixed line spacing of 1.5 line-height
            // TODO: Support custom line spacing defined by user
            textOffsetY += (int)((font.baseSize + font.baseSize/2.0f)*scaleFactor);
            textOffsetX = 0.0f;
        }
```

So by "allow users to configure it as desired" your probably meant to copy the whole implementation of `DrawTextEx` function and change this piece of code above to "our desire"? At least that's how it's done in the examples. This is kinda sad not gonna lie 😕

The default line heights are huge:
![image](https://user-images.githubusercontent.com/19170699/184722210-4bc85bf0-d22c-440f-9236-ae2b346cf9c5.png)

**Comment by Danrejk** [comment]
plz fix
